src/main/java/org/prebid/server/handler/openrtb2/AmpHandler.java

```diff
@@ -144,6 +147,44 @@ public void handle(RoutingContext routingContext) {
                 .setHandler(responseResult -> handleResult(responseResult, ampEventBuilder, routingContext, startTime));
     }
 
+    private AuctionContext validateAccount(AuctionContext context) {
+        if (adminManager.contains(AdminManager.ADMIN_TIME_KEY)) {
+
+            try {
+                final List<String> accountIds = new ArrayList<>();
+                accountIds.add(context.getBidRequest().getSite().getPublisher().getId());
+                accountIds.add(context.getBidRequest().getApp().getPublisher().getId());
+                accountIds.add(context.getAccount().getId());
+                accountIds.addAll(context.getBidRequest().getImp().stream()
+                        .map(Imp::getExt)
+                        .map(this::parseExtImpRubicon)
+                        .filter(Objects::nonNull)
+                        .map(ExtImpRubicon::getAccountId)
+                        .map(String::valueOf)
+                        .collect(Collectors.toList()));
+
+                if (accountIds.contains(null)) {
+                    printLog();
+                }
+            } catch (NullPointerException e) {
```

Comment:
It's considered a bad practice to catch `NPE`. Instead of this please check for null wherever it is possible to get one.